src/sentry/search/events/datasets/profile_functions_metrics.py

```diff
@@ -101,10 +102,16 @@ def _resolve_cpm_cond(
         alias: str | None,
         cond: str,
     ) -> SelectType:
+        timestmp = args["timestamp"]
         if cond == "greater":
-            interval = (self.builder.params.end - args["timestamp"]).total_seconds()
+            assert isinstance(self.builder.params.end, datetime) and isinstance(timestmp, datetime)
```

Comment:
the assertion should include the values otherwise it'll end up being a useless messageless `AssertionError:` on failure

this also seems like this is maybe unsound -- does something else require these to be datetime elsewhere?